lib/graphql/queries.js

```diff
@@ -162,6 +162,22 @@ export const loggedInUserQuery = gql`
             country
             structured
           }
+          children {
+            id
+            slug
+            type
+            name
+            isActive
+            imageUrl
+            host {
+              id
+            }
+            paymentMethods {
+              id
+              service
+              type
+            }
```

Comment:
I think it is important for us to keep this and, I hope that properly loading with ids, this can be cached in Apollo.
The reason is that some users can potentially have a lot of events/projects available and loading the next step to check if there's a payment method available is a bit too much.

Reply:
The only payment method we're interested in for events and projects is the account balance (service=opencollective,type=collective). Is there any reason why it wouldn't be there?

Reply:
According to https://github.com/opencollective/opencollective-frontend/pull/5893, the work we did to remove the payment methods from this query saved up to 1s for large accounts in prod. It would be sad to re-introduce this debt.

Worst case scenario, we solve it with https://github.com/opencollective/opencollective/issues/3479. But I still fail to understand why the children balance payment method would be missing.

Reply:
@Betree sure. I think this was an edge-case with broken data, after some investigation, I notice that every hosted collective should have a compatible Payment Method. I'll remove this pre-check.